### PR TITLE
Enabled conv2d batch norm fusion for traced modules.

### PR DESCRIPTION
**Pull request by kimishpatel**
Summary:
This pass enabled fusion of conv2d with batch norm for traced modules.
It requires that the module be preprocessed with freeze_module pass,
which inlines the graph constant propgates all attributes.
This pass work on top of such a graph.
However, at the moment it does not work with scripted module.
For some reason subgraph matcher is not able to find the fusion pattern,
when it operates on scripted module.

Test Plan:
python test/test_jit.py TestJit.test_foldbn_graph_mode

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

File: torch/csrc/jit/passes/graph_rewrite_helper.h [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
 namespace jit {
 namespace graph_rewrite_helper {
 
+struct ConvBNParameters {
+  at::Tensor conv_w;
+  at::Tensor conv_b;
+  at::Tensor bn_rm;
+  at::Tensor bn_rv;
+  double bn_eps = 0.0;
+  at::Tensor bn_w;
```
Comment (jerryzh168) [inline torch/csrc/jit/passes/graph_rewrite_helper.h]: could you copy these to the new file instead of moving these to helper? since this will not be used by other passes after the workaround is removed

Comment (kimishpatel) [reply torch/csrc/jit/passes/graph_rewrite_helper.h]: I cannot quite move it because it is used by BatchNormConv2dFolderHelper in quantization.cpp. I am just reusing bunch of stuff you have in quantization.cpp.

Comment (jerryzh168) [reply torch/csrc/jit/passes/graph_rewrite_helper.h]: I mean could you copy the code to the new file?

Comment (kimishpatel) [reply torch/csrc/jit/passes/graph_rewrite_helper.h]: You mean the batch norm folding in quantization.cpp? Sure, but let me do it in a different diff.
